cmd/collateral/BUILD

```diff
@@ -0,0 +1,32 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library", "go_test")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "main.go",
+    ],
+    visibility = ["//visibility:private"],
+    deps = [
+        "//cmd/client:go_default_library",
+        "//cmd/server:go_default_library",
+        "//pkg/tracing:go_default_library",
+        "@com_github_gogo_protobuf//types:go_default_library",
+        "@com_github_golang_glog//:go_default_library",
+        "@com_github_googleapis_googleapis//:google/rpc",
+        "@com_github_istio_api//:mixer/v1",
+        "@com_github_opentracing_basictracer//:go_default_library",
+        "@com_github_opentracing_opentracing_go//:go_default_library",
+        "@com_github_opentracing_opentracing_go//ext:go_default_library",
+        "@com_github_spf13_cobra//:go_default_library",
+        "@com_github_spf13_cobra//doc:go_default_library",
+        "@org_golang_google_grpc//:go_default_library",
+    ],
+)
+
+go_binary(
```

Comment:
Sorry, the PR description didn't include mixcol, I added that commit as an extra addition. I added the following comment to the source:

// This is a simple command that is used to output the auto-generated collateral
// files for the various mixer CLI commands. More specifically, this outputs
// markdown files and man pages that describe the CLI commands, along with
// bash completion files.

I sent you email over the weekend about how we can run a command within a build that was built by the current build itself. The idea was that when we build mixer, it automatically builds mixcol and then runs mixcol to produce the collateral files as part of the end-to-end build.